tool/teleport/common/teleport.go

```diff
@@ -165,6 +172,17 @@ func Run(options Options) (executedCommand string, conf *service.Config) {
 	scpc.Flag("local-addr", "local address which accepted the request").StringVar(&scpFlags.LocalAddr)
 	scpc.Arg("target", "").StringsVar(&scpFlags.Target)
 
+	// dump flags
+	dump.Flag("cluster-name",
+		"Unique cluster name, e.g. example.com.").StringVar(&dumpFlags.ClusterName)
+	dump.Flag("output",
+		"Write to stdout, default config file or custom path").Short('o').Default(
```

Comment:
Would be nice to give a couple examples of how to write to stdout, default file or provide a custom path. It's not really clear just reading the message.